public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -864,8 +864,7 @@ export class CloudWatchDatasource
     });
     if (variableName && valueVar) {
       if ((valueVar as unknown as VariableWithMultiSupport).multi) {
-        // rebuild the variable name to handle old migrated queries
-        const values = this.templateSrv.replace('$' + variableName, scopedVars, 'pipe').split('|');
+        const values = this.templateSrv.replace(value, scopedVars, 'pipe').split('|');
```

Comment:
Not introduced in the PR but in another PR related to variable migrations -  I think the name of this method is a bit misleading. It seems like it doesn't _only_ return the variable value, but also puts the value(s) in an array. Would it be possible to change the name and the comment to something that is a bit more descriptive? Maybe `expandVariableToArray`?

Reply:
done